packages/rspeedy/core/src/config/rsbuild/index.ts

```diff
@@ -50,7 +50,8 @@ export function toRsbuildConfig(
 
       polyfill: 'off',
 
-      sourceMap: config.output?.sourceMap,
+      // TODO: update the Rsbuild type to allow `sourceMap.js` to be `*-debugids` // cSpell:disable-line
```

Comment:
I'm not sure how to better handle the types, since Rsbuild is a project from a different reposity. I wanted to keep the change local to the lynx-stack.

Reply:
I'm fine with this. We could remove this when Rspack/Rsbuild updates their types